### ISSUE TITLE
Inversed Switch (Any) loses input connections when reloading workflow

### ISSUE THREAD

**Issue by stanica** [ltdrdata/ComfyUI-Impact-Pack#834]
Also happens when I press ctrl +z. 
![image](https://github.com/user-attachments/assets/71878d2e-5a2a-409c-88ff-8ca5ae8d04b2)

**Comment by ltdrdata** [comment]
Make sure Impact Pack is up to date.
latest version is v7.12

**Comment by Hullabalo** [comment]
Hi. I have the exact same problem, plus something related with 'switch any' node. I have :
- updated Impact-pack to the last version
- tried to delete and recreate new nodes
- saved as a new workflow and then re-opened
-  Comfy is up to date
- python version is 3.10.11, OS WIN 10, arch AMD64

Please, ask if you need some more infos. (and thanks for the nodes pack, anyway ;-) ) . I will do some more test in the evening, and update my comment if I find a solution.

![2024-11-30_190944](https://github.com/user-attachments/assets/d3d581fa-002c-469e-a230-e7c0d0de78cb)

**Comment by Hullabalo** [comment]
OK. Issues definitively resolved with the following steps:

Removed all 'switch any' and 'inverted switch' nodes, along with the 'select' primitive controlling the three switches (the workflow was created before the Impact Pack update. This issue does not affect workflows created after the update, as I tested).

Saved the workflow without the nodes.

Restarted Comfy and imported the workflow saved in step 2.

Recreated all the switch nodes (and, by the way, used the "select on prompt" sel_mode on all three nodes, which is better).

Saved, restarted, everything works again. 😉
